### DISCUSSION TITLE
After updating the service no longer starts

### DISCUSSION THREAD

**Discussion by alx-lat** [Stirling-Tools/Stirling-PDF#1761] [Q&A]
Hello,

Since I updated to version 0.28.3, the service won't start.

Here is the service configuration:
`  GNU nano 7.2                                                                            /etc/systemd/system/stirlingpdf.service
[Unit]
Description=Stirling-PDF service
After=syslog.target network.target

[Service]
SuccessExitStatus=143

User=root
Group=root

Type=simple

EnvironmentFile=/opt/Stirling-PDF/.env
WorkingDirectory=/opt/Stirling-PDF
ExecStart=/usr/bin/java -jar Stirling-PDF-0.28.3.jar
ExecStop=/bin/kill -15 $MAINPID

[Install]
WantedBy=multi-user.target
`

And here is the error message I get when launching the service:

`Job for stirlingpdf.service failed because of unavailable resources or another system error.

août 26 10:51:57 systemd[1]: stirlingpdf.service: Failed to load environment files: No such file or directory
août 26 10:51:57 systemd[1]: stirlingpdf.service: Failed to run 'start' task: No such file or directory
août 26 10:51:57 systemd[1]: stirlingpdf.service: Failed with result 'resources'.
août 26 10:51:57 systemd[1]: Failed to start stirlingpdf.service - Stirling-PDF service.
`

When I run this command, everything works fine though :
java -jar /opt/Stirling-PDF/Stirling-PDF-*.jar

Can you please help me?

**Comment by Frooodle** [comment]
Do your env and jar files exist with correct naming? Based on errors on something your side with file detection

**Comment by alx-lat** [comment]
Hi Frooodle,

Thanks for the quick reply!

Thanks for pointing me in the right direction.

Indeed, .env was missing from the /opt/Stirling-PDF folder.

The service now works!

I hope this topix can help others.

Thanks a lot!
